core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -87,55 +107,212 @@ final case class OptionT[F[_], A](value: F[Option[A]]) {
   def map[B](f: A => B)(implicit F: Functor[F]): OptionT[F, B] =
     OptionT(F.map(value)(_.map(f)))
 
+  /**
+   * Example:
+   * {{{
+   *  scala> import cats.data.OptionT
+   *
+   *  scala> val optionT: OptionT[List, Int] = OptionT.some[List](99)
+   *  scala> optionT.imap[Char](_.toChar)(_.toInt)
+   *  res0: OptionT[List, Char] = OptionT(List(Some(c)))
+   * }}}
+   */
   def imap[B](f: A => B)(g: B => A)(implicit F: Invariant[F]): OptionT[F, B] =
     OptionT {
       F.imap(value)(_.map(f))(_.map(g))
     }
 
+  /**
+   * Example:
+   * {{{
+   *  scala> import cats.implicits._
+   *  scala> import cats.data.OptionT
+   *
+   *  scala> type ToString[A] = Function1[A, String]
```

Comment:
This is `cats.Show` :)

Reply:
Of course :facepalm: - I was so happy I finally found an example for this I didn't even notice.
Updated.